spec/data/channel_spec.rb

```diff
@@ -103,7 +103,7 @@
         allow(JSON).to receive(:parse)
         new_data = double('new data')
         allow(new_data).to receive(:[])
-        allow(new_data).to receive(:[]).with(:permission_overwrites).and_return(false)
+        allow(new_data).to receive(:[]).with(:permission_overwrites).and_return(nil)
```

Comment:
```suggestion
        allow(new_data).to receive(:[]).with(:permission_overwrites)
```

I know the rest of this file has instances of `and_return(nil)`, but given that it's the default I would prefer not to specify it. I'm definitely open to opinion though.

Reply:
What's wrong with being more verbose than necessary ?

Personally I don't care if we keep this one or not, but why remove it and not the others in this case ?

Reply:
Just realized I never followed up on this, sorry!

I didn't mean to imply that verbosity is bad, especially in tests. It's about balance of course, and my overall preference would be for it to be consistent throughout. Anyways, after thinking about it some more I changed my mind and think it's good to go without the suggested change.